src/components/entity/state-info.js

```diff
@@ -74,10 +75,19 @@ class StateInfo extends PolymerElement {
 
         <template is="dom-if" if="[[inDialog]]">
           <div class="time-ago">
-            <ha-relative-time
-              hass="[[hass]]"
-              datetime="[[stateObj.last_changed]]"
-            ></ha-relative-time>
+            <span>
+              <ha-relative-time
+                hass="[[hass]]"
+                datetime="[[stateObj.last_changed]]"
+              ></ha-relative-time>
+              <paper-tooltip animation-delay="0">
+                [[localize('ui.dialogs.more_info_control.last_updated')]]:
```

Comment:
This string doesn't exist yet? You have to add it to `src/translations/en.json`

Reply:
added. one notice, even if string is added it doesn't show in my browser. Actually it doesn't show even if I use some other string like `ui.dialogs.more_info_control.dismiss`.